### PR TITLE
Change in runas format

### PR DESCRIPTION
**Pull request by krishnayalavarthi**
# PR Summary

'Run as Powershell 7.0' handling when invoking ps scripts, which has commands like calc.exe in some of the folder names.

## PR Context

When invoking a script with 'Run as Powershell 7.0' from a path which has commands like **calc.exe** in some of the folder names, will invoke the calc.exe and execute it. This PR will avoid such invoke.

## PR Checklist

- [ ] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [ ] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and opened an issue in the relevant tool repository. This may include:
        - [ ] Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode
        (which runs in a different PS Host).
            - [ ] Issue filed: 
        - [ ] Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
            - [ ] Issue filed: 
        - [ ] Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
            - [ ] Issue filed: 
        - [ ] Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).
            - [ ] Issue filed:

### CODE REVIEW

File: assets/Product.wxs [hunk 1]
```diff
@@ -259,7 +259,7 @@
                 <RegistryValue Type="string" Name="MUIVerb" Value="$(var.PowerShellFileContextSubMenuDialogText)"/>
                 <RegistryValue Type="string" Name="Icon" Value="[VersionFolder]pwsh.exe" KeyPath="yes"/>
                 <RegistryKey Key="Command">
-                  <RegistryValue Type="string" Value="[VersionFolder]pwsh.exe -Command &quot;$host.UI.RawUI.WindowTitle = '$(var.ProductName) $(var.SimpleProductVersion) ($(sys.BUILDARCH))'; if((Get-ExecutionPolicy ) -ne 'AllSigned') { Set-ExecutionPolicy -Scope Process Bypass }; &amp; '%1'&quot;"/>
+                  <RegistryValue Type="string" Value="[VersionFolder]pwsh.exe -file &quot; '%1'&quot;"/>
```
Comment (SteveL-MSFT) [inline assets/Product.wxs:262]: I think you don't need the inner single quotes.  The `&quot;` will add a double quote and take care of whitespace in the path

```suggestion
                  <RegistryValue Type="string" Value="[VersionFolder]pwsh.exe -file &quot;%1&quot;"/>
```

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

**Review by SteveL-MSFT — APPROVED** [review]
LGTM

**Comment by TravisEz13** [comment]
There is a conflict, and you didn't give maintainers permission to resolve it for you.  Please resolve the conflict.

**Comment by krishnayalavarthi** [comment]
**Resolve conflicts** button is disabled for me. I see a message "Only those with write access to this repository can merge pull requests."

**Comment by krishnayalavarthi** [comment]
I created a new one with the right permissions. #15434

**Comment by krishnayalavarthi** [comment]
image

Disabled 'Resolve Conflicts'

**Comment by iSazonov** [comment]
@krishnayalavarthi Since it is simple change you could update your fork, create new branch, add the change and pull new PR.

**Comment by krishnayalavarthi** [comment]
@iSazonov I actually created a new PR #15434 , still I get the conflicting files error.

**Comment by iSazonov** [comment]
> @iSazonov I actually created a new PR #15434 , still I get the conflicting files error.

You have used the same branch. I suggest you update your fork with latest commits from PowerShell/master? create new work branch and apply your changes again.

**Comment by daxian-dbw** [comment]
I just realized #15434 was created to replace this PR. I have assigned #15434 to @TravisEz13, also added @SteveL-MSFT to review again.

**Pull request closed without merging**